fixtures/local-mode-tests/tests/logging.test.ts

```diff
@@ -37,6 +38,7 @@ it("logs startup errors", async () => {
 	} catch (e) {
 		caughtError = e;
 	}
+	await setTimeout(500);
```

Comment:
Ideally a little comment here would be helpful to future readers...

```suggestion
  // wait a bit to give time for the `console` logging to complete
	await setTimeout(500);
```